recommender/engine.go

```diff
@@ -235,13 +245,43 @@ func (e *Engine) RecommendCluster(provider string, region string, req ClusterRec
 
 	cheapestNodePoolSet := e.findCheapestNodePoolSet(nodePools)
 
+	accuracy := e.findResponseSum(provider, region, req.Zones, cheapestNodePoolSet)
+
 	return &ClusterRecommendationResp{
 		Provider:  provider,
 		Zones:     req.Zones,
 		NodePools: cheapestNodePoolSet,
+		Accuracy:  accuracy,
 	}, nil
 }
 
+func (e *Engine) findResponseSum(provider string, region string, zones []string, nodePoolSet []NodePool) ClusterRecommendationAccuracy {
+	var sumCpus float64
+	var sumMem float64
+	var sumNodes int
+	for _, nodePool := range nodePoolSet {
+		sumCpus += nodePool.getSum(productinfo.Cpu)
+		sumMem += nodePool.getSum(productinfo.Memory)
+		sumNodes += nodePool.SumNodes
+	}
+
+	if zones == nil || len(zones) == 0 {
```

Comment:
the zones can be initialised/retrieved within this (if) statement
even better, zones come directly from the request / can the zones of the recommendation be detected somehow?